solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -1009,7 +1009,7 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     /// @notice Locks the state of group creation.
     /// @dev This function is meant to be used by test stubs which inherits
     ///      from this contract and needs to lock the DKG state arbitrarily.
-    function dkgLockState() internal {
+    function dkgLockState() external {
```

Comment:
Is this correct? Anyone can now lock `RandomBeacon` state even though no DKG is in progress.